feature-flags.js

```diff
@@ -0,0 +1,144 @@
+const axios = require('axios')
+const crypto = require('crypto')
+const ms = require('ms')
+const version = require('./package.json').version
+
+const LONG_SCALE = 0xfffffffffffffff
+
+class ValueError extends Error {}
```

Comment:
I don't think it makes a ton of sense to simulate Python's `ValueError` in this JS code

Reply:
Yeah, I guess so - I had a few more of these around so they at least helped the dev, but I guess now it doesn't make sense, will remove